includes/Commerce/Orders.php

```diff
@@ -69,10 +82,84 @@ public function create_local_order( Order $remote_order ) {
 	 * @param Order $remote_order Orders API order object
 	 * @param \WC_Order $local_order local order object
 	 * @return \WC_Order
+	 * @throws SV_WC_Plugin_Exception|\WC_Data_Exception
 	 */
 	public function update_local_order( Order $remote_order, \WC_Order $local_order ) {
 
-		// TODO: implement
+		// add/update items
+		foreach ( $remote_order->get_items() as $item ) {
```

Comment:
It looks like this PR and #1515 contain the same changes, both to `find_local_order()` and `update_local_order()`. Can we get the two isolated to their respective methods?

Reply:
Weird. I force-pushed this one to the correct commit.